docs/wagtail-pages.md

```diff
@@ -0,0 +1,211 @@
+[rich text box with the structure of individual components](https://torchbox.com/blog/rich-text-fields-and-faster-horses/).
+
+For example, our `LandingPage` page model 
+[includes a `header` StreamField](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/landing_page.py) 
+that can have a hero and/or a text introduction:
+
+```python
+from wagtail.wagtailcore.fields import StreamField
+
+from v1.atomic_elements import molecules
+from v1.models import CFGOVPage
+
+
+class LandingPage(CFGOVPage):
+
+    header = StreamField([
+        ('hero', molecules.Hero()),
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+```
+
+The specifics of StreamField block components can be found in 
+[Creating and Editing Wagtail Components](https://cfpb.github.io/cfgov-refresh/editing-components/).
+
+## Panels
+
+Editor panels define how the page's [fields](#fields) and [StreamFields](#streamfields) will be organized for content editors 
+and correspond to the tabs that appear across the top of the edit view for pages in the Wagtail admin.
+
+The base Wagtail `Page` class and the [`v1.models.base.CFGOVPage` class](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L60) 
+define a specific set of panels to which all fields should be added:
+
+- `content_panels`:
+  For page body content.
+  These fields appear on the "General Content" tab when editing a page. 
+- `sidefoot_panels`:
+  For page sidebar or footer content.
+  These fields appear on the "Sidebar" tab when editing a page. 
+- `settings_panels`:
+  Page configuration such as the categories, tags, scheduled publishing, etc.
+  Appears on the "Configuration" tab when editing a page. 
+
+Most fields will simply require a `FieldPanel` to be added to one of the sets of panels above. 
+StreamFields will require a `StreamFieldPanel`. 
+See the [Wagtail documentation for additional, more complex panel options](https://docs.wagtail.io/en/v1.13.4/topics/pages.html#editor-panels).
+
+For example, in our `BrowsePage` (used in the [database fields example above](#fields)),
+the `secondary_nav_exclude_sibling_pages` `BooleanField` 
+[is added to the `sidefoot_panels` as a `FieldPanel`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/browse_page.py#L63):
+
+```python
+from django.db import models
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+from v1.models.base import CFGOVPage
+
+
+class BrowsePage(CFGOVPage):
+    secondary_nav_exclude_sibling_pages = models.BooleanField(default=False)
+
+    # …
+
+    sidefoot_panels = CFGOVPage.sidefoot_panels + [
+        FieldPanel('secondary_nav_exclude_sibling_pages'),
+    ]
+```
+
+Because `secondary_nav_exclude_sibling_page` is a boolean field, 
+this creates a checkbox on the "Sidebar" tab when editing a page.
+Checking or unchecking that checkbox will set the value of `secondary_nav_exclude_sibling_pages` when the page is saved.
```

Comment:
```suggestion
Checking or unchecking that checkbox will set the value of `secondary_nav_exclude_sibling_pages` when the page is saved.
```

Reply:
What's the change here? I don't see it in the suggestion.

Reply:
I guess this was a mistake.